app/src/main/res/values/strings.xml

```diff
@@ -459,12 +459,19 @@
     <string name="wallpaper_download_error_snackbar_action">Try again</string>
     <!-- Snackbar message for when wallpaper couldn't be selected because of the disk error -->
     <string name="wallpaper_select_error_snackbar_message">Couldn’t change wallpaper</string>
-
+    <!-- Text displayed that links to website containing documentation about the "Limited Edition" wallpapers. -->
+    <string name="wallpaper_learn_more">Learn more</string>
     <!-- Label for switch which toggles the "tap-to-switch" behavior on home screen logo -->
     <string name="wallpaper_tap_to_change_switch_label_1" moz:removedIn="105" tools:ignore="UnusedResources">Change wallpaper by tapping Firefox homepage logo</string>
     <!-- This is the accessibility content description for the wallpapers functionality. Users are
     able to tap on the app logo in the home screen and can switch to different wallpapers by tapping. -->
     <string name="wallpaper_logo_content_description" moz:removedIn="105" tools:ignore="UnusedResources">Firefox logo - change the wallpaper, button</string>
+    <!-- Text for classic wallpapers title. -->
+    <string name="wallpaper_classic_title">Classic Firefox</string>
+    <!-- Text for limited edition wallpapers title. -->
+    <string name="wallpaper_limited_edition_title">Limited Edition</string>
+    <!-- Description text for the limited edition wallpapers. -->
+    <string name="wallpaper_limited_edition_description">The new Independent Voices collection.. </string>
```

Comment:
```suggestion
    <string name="wallpaper_limited_edition_description">The new Independent Voices collection.</string>
```
This still isn't correct. You want a string with "The new Independent Voices collection. %s" to append the "Learn more".